core/arch/arm/plat-ti/platform_config.h

```diff
@@ -45,8 +45,10 @@
 
 #if defined(PLATFORM_FLAVOR_dra7xx)
 #define CONSOLE_UART_BASE       UART1_BASE
-#elif
+#elif defined(PLATFORM_FLAVOR_am57xx)
 #define CONSOLE_UART_BASE       UART3_BASE
+#else
+#error "Unknown platform flavor"
```

Comment:
@glneo right, I've externally specified `PLATFORM_FLAVOR_am57xx` here just for readability purposes and maybe for a case if additional flavor to the upper `#ifdef` condition will be added (`#ifdef defined(PLATFORM_FLAVOR_dra7xx) || defined(PLATFORM_FLAVOR_am57xx)`), for not forgetting to add a specific UART base for this flavor. I know that for DRA7xx/AM57xx config it's low probability of such changes, but anyway :)